boilerplate/app/index.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Vaca App</title>
+</head>
+
+<body ng-app="VacaApp">
+
+    <main  ng-controller="VacaController as vacactrl">
+
+      <pre>{{vacactrl.vaca}}</pre>
+      <vaca-note text="vacactrl.message" file="koala"></vaca-note>
```

Comment:
I don't think this is doing anything.  You are not returning anything to put in this space.